core/src/main/java/hudson/Functions.java

```diff
@@ -2297,4 +2297,14 @@ public static boolean isContextMenuVisible(Action a) {
             return true;
         }
     }
+
+    /**
+     * @return Time interval in seconds between two ajax call, minimum and default is 5.
```

Comment:
Forcing an arbitrary minimum seems to be insulting the intelligence of users. If I know about a system property, trust me to use set it.